.github/workflows/test.yml

```diff
@@ -15,13 +15,13 @@ jobs:
       fail-fast: false
       matrix:
         sage-version:
-          - '9.2'
-          - '9.3'
```

Comment:
We may want to keep these but I'm not sure how far back we should be testing. Do you know if Sage has an official deprecation/support policy for older versions?